docs/readmes/proposals/p027_magma_outbound_roaming_support.md

```diff
@@ -0,0 +1,136 @@
+
+## 3 Implementation
+
+### 3.1 Diagrams
+
+The Roaming Gateway diagram shows the services which will run in the Roaming gateway. These services will support the user plane data transport for both inbound roaming GTP aggregation and outbound roaming when traffic is home routed.
+
+![roaming gateway](https://user-images.githubusercontent.com/93994458/154709967-7cd453ba-2070-4733-9e3f-dcc6f38624d2.png)
+
+Overall, when combined with the orchestrator and federation gateway, the proposed architecture is depicted below showing all key interfaces for inbound and outbound roaming.
+
+![arch](https://user-images.githubusercontent.com/93994458/154710530-f535e6c1-8221-4de4-9e7b-5fc974f2ee3d.png)
+
+The proposed architecture gives the Magma deploying operator the ability to locate the user plane carrying Roaming Gateway outside of AWS where data transport rates are more competitive and/or in a location favorable for connection to the GRX/IPX network.
+
+### 3.2 Scope of Change
+
+#### 3.2.1 Roaming Gateway Services
+
+The following existing services will be ported to run in the Roaming Gateway:
+
+- Health
+- S8_proxy
+- PolicyDB
+- SessionD
+- PipelineD
+- MobilityD
+- SubscriberDB
+- Control Proxy
+- Magmad
+- Eventd
+- td-agent-bit
+
+Similar to FeG and AGW, Roaming Gateway will be a full Magma gateway with control proxy connection to orchestrator and health service.
+
+##### 3.2.1.1 Health (modified)
+
+In addition to the health metrics added in the GTP Gateway proposal for inbound roaming, the following health metrics will also be added to health:
+
+- S8-U Inbound Traffic Rate (Mbps)
+- S8-U Outbound Traffic Rate (Mbps)
+- S8-U Inbound Bytes
+- S8-U Outbound Bytes
+- Active Inbound Sessions
+- Active Outbound Sessions
+
+##### 3.2.1.2 PolicyDB + SessionD + PipelineD + MobilityD(modified)
+
+The combination of policyDB, sessiond, and pipelined provide the PCEF functionality to the Roaming gateway for home routed data sessions. This ensures that policy rules, QoS profiles, Rating Groups, and charging can be applied to subscriber sessions while getting services from a VPLMN.
+Once the GTP-C session is created on FeG, Roaming Gateway sessiond process will be notified via gRPC and OVS flow will be installed to enable GTP-U tunnel and start sending data to the SGi interface of the Roaming Gateway associated with the session APN.  Having sessiond manage the tunnel lifecycle mirrors how it is done in the case of 5G in AGW and should reuse code where possible.
+
+##### 3.2.1.3 MobilityD
+
+UE IP address allocation will be provided by MobilityD. The UE IP request will be moved from MME function to sessiond to provide consistency across Roaming Gateway and AGW. IP allocation via NAT will be supported with a configurable NAT pool for IPs. Other IP address allocation schemes are out of scope.
+
+In this phase, default bearer and general internet access is provided to home routed sessions. Dedicated bearer and other services are out of scope.
```

Comment:
What do you mean by `In this phase`? Maybe `currently` was meant? Or is it part to be implemented?
The statement should be made unambiguous.